servant-client/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.4.1
+-----
+* `Delete` allow zero `'[]` or more `'[JSON]` content types on delete
```

Comment:
This is a little strangely worded. I find it confusing.

Do you mean something like:

  The `HasClient` instance for `Delete cts ()` now does not care at all about content types provided.